content/blog/2020-announce-ticket-sales.md

```diff
@@ -0,0 +1,47 @@
+Title: PyCon India 2020 Tickets are going live soon! Grab yours before the special offer runs out!
```

Comment:
What is special offer?

Reply:
Ehh, Early Bird I thought?

Reply:
Okay! Maybe `grab those few Early Bird tickets`?

Reply:
Removed, resolving.